connectors/src/lib/nango_client.ts

```diff
@@ -48,6 +48,10 @@ class CustomNango extends Nango {
       throw e;
     }
   }
+
+  async listConnections() {
+    return await super.listConnections();
```

Comment:
Why is that needed if we only call super.listConnections()?

Reply:
Because `nango_client()` that is used everywhere is CustomNango, and I think it's cleaner to reuse the same pattern as everywhere else 🙏🏻